### PR TITLE
Remove dependency on simplejson package.

### PR DESCRIPTION
**Pull request by mcepl**
I don't think anybody needs compatiblity with Python 2.6 any more.

The only difference in API of the standard library json methods and the ones from simplejson is attribute `encoding`, which has been removed in the standard library (for good reasons). On the other hand, the standard library now accepts both `str` and `bytes`, so encoding is not necessary that much (and if it is necessary, it should be done in the class specific encoder anyway).

The only change in API is that `web.pgadmin.utils.ajax.make_json_response()` function lost its `encoding` parameter. It is never used in all pgadmin4 anyway and given that `DataTypeJSONEncoder` produces pure Python `str` anyway, I don't think it is a big loss.

### CODE REVIEW

**Comment by khushboovashi** [comment]
@mcepl, 

Looks good to me, only some of the files/modules are still left like clouds, publications etc.
Also, remove encoding keyword argument from the make_json_response in sqleditor/__init__py file. Please check another instances too for the same.

Thanks,
Khushboo

**Comment by mcepl** [comment]
> Looks good to me, only some of the files/modules are still left like clouds, publications etc. Also, remove encoding keyword argument from the make_json_response in sqleditor/__init__py file. Please check another instances too for the same.

Whoops, I missed those. Fixed.

**Comment by khushboovashi** [comment]
> > Looks good to me, only some of the files/modules are still left like clouds, publications etc. Also, remove encoding keyword argument from the make_json_response in sqleditor/__init__py file. Please check another instances too for the same.
> 
> Whoops, I missed those. Fixed.

Thanks for the quick fix, but I still get error (make_json_response() got an unexpected keyword argument 'encoding') while trying to execute the query in the Query Tool.  So, you need to remove the 'encoding' parameter from make_json_response function in sqleditor/__init__.py file.

**Comment by mcepl** [comment]
Those call were fixed. Are those attributes `encoding` in `test_import_export_create_job_unit_test.py` and `test_backup_create_job_unit_test.py`? Should I remove them as well?

**Comment by khushboovashi** [comment]
> Those calls were fixed. Are those attributes `encoding` in `test_import_export_create_job_unit_test.py` and `test_backup_create_job_unit_test.py`? Should I remove them as well?

No, They are required, so do not remove those. 

With this patch, I am facing one issue, while saving/updating the non ascii data from the Sql Editor, I am getting the error saying  "Object of type bytes is not JSON serializable".

You can reproduce this issue by creating  a database with latin9 or latin8 (other than UTF8) encoding. Then try to save the non-ascii characters (ex: äöüßs).

**Comment by mcepl** [comment]
> With this patch, I am facing one issue, while saving/updating the non ascii data from the Sql Editor, I am getting the error saying "Object of type bytes is not JSON serializable".

If there is a need for explicit encoding (which object have allowed non-UTF8 content?) with `.encode()/.decode()` methods of `str` and `bytes`.

**Comment by khushboovashi** [comment]
> > With this patch, I am facing one issue, while saving/updating the non ascii data from the Sql Editor, I am getting the error saying "Object of type bytes is not JSON serializable".
> 
> If there is a need for explicit encoding (which object have allowed non-UTF8 content?) with `.encode()/.decode()` methods of `str` and `bytes`. Where did the crash happened?

Try the steps which I have mentioned to reproduce it.

**Comment by mcepl** [comment]
> Try the steps which I have mentioned to reproduce it.

I have to admit a shameful thing: I have nothing to do with PostgreSQL, I don’t have it even installed. I am a maintainer of Python packages for SUSE/openSUSE (all 2000+ of them, together with two more colleagues), and this PR is driven mostly by my intent to get rid of `simplejson` package in our distro, I know almost nothing about PostgreSQL.
